src/test/java/com/zorranlabs/interview/espn/EspnSeleniumTest2.java

```diff
@@ -0,0 +1,42 @@
+package com.zorranlabs.interview.espn;
+
+import junit.framework.TestCase;
+import org.hamcrest.CoreMatchers;
+import org.junit.Assert;
+import org.openqa.selenium.*;
+import org.openqa.selenium.firefox.FirefoxDriver;
+
+public class EspnSeleniumTest2 extends TestCase {
+    private WebDriver driver;
+
+    public void setUp() throws Exception {
+        driver = new FirefoxDriver();
+    }
+
+    public void testSimple() throws Exception {
+        // WebDriver driver;
+        driver.get("http://espn.go.com");
+
+        // Find a link
+        WebElement nfl_scores_link = driver.findElement(By.cssSelector("a[href*='http://scores.espn.go.com/nfl/scoreboard']"));
+        Assert.assertTrue("http://scores.espn.go.com/nfl/scoreboard link is not found", null != nfl_scores_link);
+        System.out.println("Scoreboard link: " + nfl_scores_link.toString());
+        nfl_scores_link.click();
+
+        // Get and assert title
+        String sScoreboardTitle = driver.getTitle();
+        System.out.println("Page title is: " + sScoreboardTitle);
+        Assert.assertThat("Verifying scoreboard title", sScoreboardTitle, CoreMatchers.containsString("NFL Scoreboard"));
```

Comment:
Are you talking about using code like this:
import static org.junit.Assert.assertTrue; 
and use it in code as 
assertTrue("http://scores.espn.go..... );?

If so, I know about this. However, I use to work a lot with JUnit3, even these days, extend JUnit3 based tests cases. In JUnit3 methods like assertEquals inherited since the test class extended junit.framework.TestCase. Only in JUnit4, test classes no longer need to extend TestCase and can statically import the assert methods from org.junit.Assert. 
However, habit (even bad one) is habit. So, to me is easier to explicitly mention it.